L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigisPostLS1_cfi.py

```diff
@@ -1,5 +1,10 @@
 import FWCore.ParameterSet.Config as cms
 
+print " ######################################################################################################### "
```

Comment:
please use sys.stderr.write() instead of print , to avoid the message ending up inside the actual python dumps